### DISCUSSION TITLE
Any way to disable number merging in C#.NET and English Culture?

### DISCUSSION THREAD

**Discussion by vignatov** [microsoft/Recognizers-Text#2497] [Q&A] [answered]
The current system joins the numbers. For example if a string goes like five twenty six, it returns 5 and 26. Any way to stop it from joining 20 and 6 together, so the output be 5 20 6?

**Comment by tellarin** [comment, marked as answer]
Unofrtunately this functionality is not available and not in the planned roadmap.

**Comment by martinmueller4voice** [comment]
You could split the input into words and then handle each word separately. This should do the trick.

hth
mav
